src/app/body/profile/edit-skills/edit-skills.component.html

```diff
@@ -24,7 +24,7 @@ <h2 class="modal-title" id="exampleModalLabel">Your Skills</h2>
                 </div>
                 <div class="col-md-1 col-2 text-right ">
                     <span class="btn">
-                        <mat-icon fontSet="material-icons-outlined" data-dismiss="modal"class="align-middle "(click)="editSkillCompleted()" title="Close">
```

Comment:
the method should not be removed, it should be given to emit the skills to parent properly.